docs/attendances_and_absences.md

```diff
@@ -0,0 +1,148 @@
+      - if a `ServiceDay` does not exist for that day for that child
+        - create a `ServiceDay` for that day for that child w/ the `absence_type`
+    - if the child is not scheduled for that day
+      - if a `ServiceDay` exists for that day for that child
+        - delete the `ServiceDay`
+      - if a `ServiceDay` does not exist for that day for that child
+        - do nothing
+
+    > Behavior currently not defined 
+    > - error handling
+
+#### _Daily Wonderschool CSV Import - `Wonderschool::Necc::AttendanceCsvImporter`_
+
+**Assumption: We do not get absences from Wonderschool**
+
+- for each row:
+  - if an attendance exists with the `wonderschool_id`
+    - update the attendance w/ the `check_in`, `check_out`
+  - if an attendance does not exist with the `wonderschool_id`
+    - if a `ServiceDay` exists for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out`, `wonderschool_id` associated to that `ServiceDay`
+    - if a `ServiceDay` does not exist for that day for that child
+      - create a `ServiceDay` for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out`, `wonderschool_id` associated to the new `ServiceDay`
+
+  > Behavior currently not defined 
+  > - if an attendance exists in the wonderschool import one day and does not exist the next day, we do not find missing attendances and remove them
+  > - there is currently no programmatic or user-facing way to do delete Wonderschool attendances
+  > - error handling
+
+#### _User-facing UI Attendance & Absence Creation - `AttendanceBatchesController#create` & `ServiceDaysController#create`_
+
+- for each record in the batch:
+  - if the record contains an attendance
+    - if a `ServiceDay` exists for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out` associated to that `ServiceDay`
+    - if a `ServiceDay` does not exist for that day for that child
+      - create a `ServiceDay` for that day for that child
+      - add an `Attendance` w/ the `check_in`, `check_out`, associated to the new `ServiceDay`
+  - if the record contains an absence
+    - if the child is scheduled for that day
+      - if a `ServiceDay` exists for that day for that child
+        - change the `ServiceDay` to an absence w/ the `absence_type`
+      - if a `ServiceDay` does not exist for that day for that child
+        - create a `ServiceDay` for that day for that child w/ the `absence_type`
+    - if the child is not scheduled for that day
+      - if a `ServiceDay` exists for that day for that child
+        - delete the `ServiceDay`
+      - if a `ServiceDay` does not exist for that day for that child
+        - do nothing
+
+  > Behavior currently not defined 
+  > - error handling
+
+### Daily Automatic Absence Creation
+
+> `Nebraska::AbsenceGenerator`
+
+- for all children in Nebraska, generates an absence if there's no current attendance AND if the child is scheduled that day
```

Comment:
@rebeccakarasiktw @katelovescode we need to decide if we should 
1- keep this as is
2- get rid of this logic entirely
3- change this logic somehow as a part of the CSV import / WS import

Reply:
Per discussion with @katelovescode we will keep as is but change the absence type. We may later change the rules around this.